Modules/Loadable/VolumeRendering/Logic/vtkSlicerVolumeRenderingLogic.cxx

```diff
@@ -1173,3 +1173,19 @@ void vtkSlicerVolumeRenderingLogic::RemovePreset(vtkMRMLVolumePropertyNode* pres
     }
   presetScene->RemoveNode(preset);
 }
+
+//---------------------------------------------------------------------------
+int vtkSlicerVolumeRenderingLogic::LoadCustomPresetsScene(const char* sceneFilePath)
```

Comment:
This helper function is not really needed, as you can call clear the scene by simply calling

    logic->GetPresetsScene()->Clear();

or import a scene as described here: https://www.slicer.org/wiki/Documentation/Nightly/Modules/VolumeRendering#How_to_register_custom_volume_rendering_presets

but I guess this should work, too:

    logic->GetPresetsScene()->SetURL(...);
    logic->GetPresetsScene()->Import();

If you think helper functions are needed then it would be better to keep the name and behavior of the called methods. For example, instead of LoadCustomPresersScene() it would be better to name it ImportCustomPresetsScene().

Reply:
I don't see why having a utility function for something is undersired. We have tons of functions that save the developers writing a few lines of code.

"Import" implies that whatever was in the presets scene before stays. This convenience function however imports the file to an empty scene. So the name "load" was intentional.